### PR TITLE
Some more documentation updates

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

**Comment by poettering** [comment]
looks all good to me, but the journald one kinda covers the same stuff as my #6839 i posted 30min earlier ;-)

how do we proceed regarding this? maybe you want to pick of the one commit from #6839 and integrate it in your patch set?

**Comment by keszybz** [comment]
Yeah, need to coordinate better. Usually it's not a problem, but now when we're working off a short list, it's likely. I'll assign each issue or PR I'm working on or reviewing to myself, and I'd appreciate if others do the same.

> maybe you want to pick of the one commit from #6839 and integrate it in your patch set?

Will do.

**Comment by keszybz** [comment]
Updated, including your commit.

**Comment by poettering** [comment]
hmm, this doesn't look right, somehow keyring related commits now ended up here?

**Comment by keszybz** [comment]
Wrong branch. Updated.

**Comment by poettering** [comment]
lgtm.

File: man/systemd-journald.service.xml [hunk 1]
```diff
@@ -112,6 +111,49 @@ systemd-tmpfiles --create --prefix /var/log/journal</programlisting>
...
+    their metadata, but will continue to describe the original process. This is different from the other logging
+    transports listed above, which are inherently record based and where the metadata is always associated with the
+    individual record.</para>
+
+    <para>In addition to the the implicit standard output/error logging of services, stream logging is also available
+    via the <citerefentry><refentrytitle>systemd-cat</refentrytitle><manvolnum>1</manvolnum></citerefentry> command
+    line tool.</para>
+
+    <para> Currently, the number of parallel log streams <filename>systemd-journald</filename> will accept is limited
+    to 4096.</para>
```
Comment (evverx) [inline man/systemd-journald.service.xml:153]: I'm not sure whether it should be mentioned or not, but if a service starts when the limit has already been reached, it receives `EPIPE` right from the start even though `journald` is never stopped.

Comment (poettering) [reply man/systemd-journald.service.xml:153]: let's add that in a later PR.

**Pull request merged**